packages/common-ethereum/src/project/models.ts

```diff
@@ -152,7 +149,7 @@ export class EthereumMapping implements SubqlMapping {
   })
   @IsArray()
   @ValidateNested()
-  handlers: SubqlHandler[];
+  handlers: SubqlRuntimeHandler[];
```

Comment:
This was the right type before. With this change we remove support for custom handlers.